langs/en.json

```
@@ -0,0 +1,152 @@
+{
+  "site.title": "Ultramarine Linux",
+  "site.title.short": "Ultramarine",
+  "site.description": "Ultramarine Linux is a Fedora-based Linux distribution designed to stay out of your way and be easy to use. All editions come with several tweaks preapplied to make initial setup and daily usage seamless.",
```

```suggestion
  "site.description": "Ultramarine is a modern operating system that lets you just do your thing with the hardware and tools you already love.",
```
to match the new desc, idk why this didn't get updated